src/js/components/Chart/index.d.ts

```diff
@@ -8,14 +8,14 @@ export interface ChartProps {
   margin?: MarginType;
   bounds?: number[][];
   color?: string | {color?: string,opacity?: "weak" | "medium" | "strong" | boolean};
-  onClick?: ((...args: any[]) => any);
+  onClick?: ((event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void);
```

Comment:
I'm thinking this should be extracted and reused, to avoid duplications

Reply:
Hi Shimi -- I agree, I could extract it to a single onClickType variable. I'll also attach a markup